build/Dockerfile

```
@@ -0,0 +1,17 @@
+FROM golang:1.15.2-buster
+
+ARG GH_VERSION='1.4.0'
```

One can never be up-to-date enough
```suggestion
ARG GH_VERSION='1.5.0'
```